docs/collections/catalog/prefect-fivetran.yaml

```diff
@@ -0,0 +1,6 @@
+collectionName: prefect-fivetran
+author: Fivetran
+authorUrl: https://prefect.io
```

Comment:
This needs to be updated to point to fivetran as maintainer and a valid link to either docs or the repo